modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1153 @@
+
+=== New togglebutton in dialog footer buttons
+//#TINY-9523
+
+The {productname} modal footer and view header dialog have recently added a new improvement by introducing a new togglebutton. This togglebutton allows users to quickly and easily toggle between the footer buttons withing the dialog. This makes navigation easier and more efficient than having to manually search through menus and dropdowns. 
+
+The togglebutton is present in {productname} 6.4.1 and it helps users to simplify their navigation of the footer buttons dialog. With the introduction of this togglebutton, users are now able to access footer buttons in a more efficient manner. The new togglebutton is a great addition to the {productname} modal footer and view header dialog, as it greatly simplifies the navigation of the footer buttons dialog and increases the overall usability of the feature.
+
+=== Added toggleFullscreen to dialog API
+//#TINY-9528
+
+{productname} recently added a new *"fullscreen"* toggle button to the Advanced Code Editor Premium plugin, that gives users the ability to expand the Advanced Code Editor dialog toolbar in order to make better use of the available space on their screen, while providing them with a more comfortable working environment.
+
+The toggle button is easily accessible for users that have the Advanced Code Editor Premium plugin by integrating it into their editor by using the `plugins: [ "fullscreen" ]` in pair with the below options.
+
+.Example of fullscreen mode integration within {productname}.
+[source, js]
+----
+tinymce.init({
+  selector: "textarea",
+  advcode_inline: true,
+  plugins: [
+    "code", "fullscreen", 'advcode',
+  ],
+  toolbar: "code"
+});
+----
+
+As a result, the toggle button can be found in the top left corner of the Advanced Code Editor Premium toolbar. Once clicked, it will expand the dialog toolbar to fill the entire window, providing a more comprehensive view of the available options. A second click of the toggle button will return the dialog toolbar to its original size.
+
+This option is now available in {productname} 6.4.1, as part of the core/ui API.
+
+=== New text-size-increase and text-size-decrease icons
+//#TINY-9530
+
+The {productname} package files have been updated with two new icons that enable users to increase or decrease the size of text in the Advanced Code Editor. The new icons are the `text-size-increase` and `text-size-decrease` icons. These new icons have been added to make it easier for users to adjust the size of the text within the code editor while making it easier to read and understand the code they are writing. 
+
+.Example of `textSizeIncrease` and `textSizeDecrease` integration within {productname}.
+[source, js]
+----
+tinymce.init({
+  selector: "textarea",
+  advcode_inline: true,
+  plugins: [
+    "code", "advcode", 
+  ],
+  toolbar: "textSizeIncrease textSizeDecrease"
+});
+----
+[NOTE]
+This plugin feature is only available for paid {productname} subscriptions.
```

Comment:
```suggestion
```
Don't need to document new icons, and they're only included in the advanced code dialog, not available as toolbar buttons.